geoportailv3/static/js/mymaps/mymaps.html

```diff
@@ -1,11 +1,63 @@
-<div><!-- TODO block to be shown only in cases where no map is selected -->
+<div ng-show="!ctrl.mymap">
   <p translate>
     Create, save and share your own maps.
   </p>
-  <p class="text-center" ng-if="mainCtrl.loggedin">
-    <button class="btn btn-default" translate>Open a map</button>
+  <p class="text-center"><!-- Show only when logged in -->
+    <button class="btn btn-default" ng-click="ctrl.choose()" translate>Open a map</button>
```

Comment:
If it was me I'd use more explicit method names, like `chooseMap` and `createMap`. I think it's obvious when you're developing the functionality, but it will be less obvious to a new developer.